UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API.FunctionalTests/FunctionalTests/ExchangeSetAfterSpecifiedDateTime.cs

```diff
@@ -12,18 +12,63 @@ class ExchangeSetAfterSpecifiedDateTime
     {
         private ExchangeSetApiClient ExchangesetApiClient { get; set; }
         private TestConfiguration Config { get; set; }
+        private string EssJwtToken { get; set; }
+        private string EssJwtTokenNoRole { get; set; }
+        private string EssJwtCustomizedToken { get; set; }
 
         [SetUp]
-        public void Setup()
+        public async Task SetupAsync()
         {
             Config = new TestConfiguration();
             ExchangesetApiClient = new ExchangeSetApiClient(Config.EssBaseAddress);
+            AuthTokenProvider authTokenProvider = new AuthTokenProvider();
+            EssJwtToken = await authTokenProvider.GetEssToken();
+            EssJwtTokenNoRole = await authTokenProvider.GetEssTokenNoAuth();
+            EssJwtCustomizedToken = authTokenProvider.GenerateCustomToken();
         }
+
         [Test]
-        public async Task WhenICallTheApiWithAValidRFC1123DateTime_ThenACorrectResponseIsReturned()
+        public async Task WhenICallTheApiWithOutAuthToken_ThenAnUnauthorisedResponseIsReturned()
         {
             string sincedatetime = "Mon, 01 Mar 2021 00:00:00 GMT";
             var apiresponse = await ExchangesetApiClient.GetExchangeSetBasedOnDateTimeAsync(sincedatetime);
+            
+            Assert.AreEqual(401, (int)apiresponse.StatusCode, $"Incorrect status code {apiresponse.StatusCode} is returned, instead of the expected 401.");
+        }
+
+        [Test]
+        public async Task WhenICallTheApiWithTamperedToken_ThenAnUnauthorisedResponseIsReturned()
+        {
+            string sincedatetime = "Mon, 01 Mar 2021 00:00:00 GMT";
+            string TamperedEssJwtToken = EssJwtToken.Remove(EssJwtToken.Length - 2);
+            var apiresponse = await ExchangesetApiClient.GetExchangeSetBasedOnDateTimeAsync(sincedatetime, accessToken: TamperedEssJwtToken);
```

Comment:
Please can we have variable name casing consistent across all code.  
```suggestion
            string sinceDatetime = "Mon, 01 Mar 2021 00:00:00 GMT";
            string tamperedEssJwtToken = EssJwtToken.Remove(EssJwtToken.Length - 2);
            var apiResponse = await ExchangesetApiClient.GetExchangeSetBasedOnDateTimeAsync(sincedatetime, accessToken: TamperedEssJwtToken);
```

Reply:
Made changes for variable name casing consistent across all code. Thanks Vijendra.